### ISSUE TITLE
Remember fullscreen state of videos when (auto)playing next

### ISSUE THREAD

**Issue by ShaunaGordon** [nothingworksinc/ticketbeast#68]
When moving to the next video with autoplay on, the video drops out of fullscreen. It would be nice if it stayed or went back to fullscreen, so that the user doesn't have to keep switching to it and making it fullscreen.

Use case: I have the course playing on my TV via Chromecast (casting the tab). I fullscreen the player so that it covers the whole screen and I can see the code being written.

When the next video starts, it's back to the "embedded" view and have to go onto my computer, switch to the tab, and make it fullscreen again. It would be better if the video stayed full screen without my intervening.

**Comment by TiborBesze** [comment]
Yeah, that would be great. :) Also, it would be really nice, if we had YouTube flavoured keyboard shortcuts, like:
F to enter/exit full screen
J to seek backwards X seconds
K to play/pause
L to seek forward X seconds

**Comment by vesper8** [comment]
Would also love if it stayed in full screen

I like that you can use the arrow keys to jumping backward/forward. Some additional keyboard shortcuts would be cool to have

If staying in fullscreen is very difficult.. an alternative would be to have a single video with all lessons maybe
